sdk/formrecognizer/ai-form-recognizer/src/formTrainingClient.ts

```diff
@@ -130,6 +130,10 @@ export type BeginTrainingOptions = TrainingFileFilter & {
   updateIntervalInMs?: number;
   onProgress?: (state: TrainingOperationState) => void;
   resumeFrom?: string;
+  /**
+   * An optional name to associate with the model
+   */
+  modelName?: string;
```

Comment:
in .NET we are moving this out of the option bag and leaving it as a parameter of BeginTraining becuase we expect customer to use model name frequently.
Could this be the case for JS too?